pkg/client-sdk/Makefile

```diff
@@ -31,4 +31,15 @@ GO_VERSION := $(shell go version | cut -d' ' -f3)
 build-wasm:
 	@mkdir -p $(BUILD_DIR)
 	@echo "Version: $(VERSION)"
-	@GOOS=js GOARCH=wasm GO111MODULE=on go build -ldflags="-X 'main.Version=$(VERSION)'" -o $(BUILD_DIR)/ark-sdk.wasm $(WASM_DIR)/main.go
+	@GOOS=js GOARCH=wasm GO111MODULE=on go build -ldflags="-s -w -X 'main.Version=$(VERSION)'" -o $(BUILD_DIR)/ark-sdk.wasm $(WASM_DIR)/main.go
+
+test-wasm:
+	$(MAKE) build-wasm BUILD_DIR=./test/wasm/web
+	@echo "Copying $(go env GOROOT)/misc/wasm/wasm_exec.js"
+	@cp `go env GOROOT`/misc/wasm/wasm_exec.js ./test/wasm/web
+	@echo "Starting web server..."
+	@cd ./test/wasm/web && python3 -m http.server 8000 &
+	@echo "Waiting for server to start..."
+	@sleep 3
+	@echo "Running tests..."
+	@cd ./test/wasm && go test -v
```

Comment:
once the test is done (either succeded or failed) we must clean folders and stop services

Reply:
why? isnt GHA container destroyed after pipeline runs? i dont see we stop nigiri for eg in integration pipeline?